src/main/java/io/pivotal/cla/service/github/MylynGitHubApi.java

```diff
@@ -175,6 +177,7 @@ private boolean hasObviousFixComment(Collection<? extends Comment> comments, Str
 		Optional<? extends Comment> obviousFixComment = comments.stream() //
 				.filter(comment -> comment.getUser() != null && !claUserLogin.equals(comment.getUser().getLogin())) //
 				.filter(comment -> hasObviousFix(comment.getBody())) //
+				.filter(comment -> comment.getBody().startsWith("@" + claUserLogin + " ")) //
```

Comment:
I wonder if it makes more sense to update `hasObviousFix` method rather than adding an additional filter here?

Reply:
I started by doing that, but then this method also covers the PR body, which is a bit of a different case in my opinion since it is less likely the author would know to mention the bot from the PR's inception. But as I pondered in the issue, I wonder if it would make sense to stop watching the PR body altogether.

Reply:
I think it does make sense as we often get repeat contributors that might include it in the body.

Reply:
so explicitly detect the string `"@bot this is an obvious fix"`?

Reply:
Yes. I think so